python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -103,6 +104,10 @@ class DetectAndMeasureConfig(pipeBase.PipelineTaskConfig,
         target=SourceDetectionTask,
         doc="Final source detection for diaSource measurement",
     )
+    deblend = pexConfig.ConfigurableField(
```

Comment:
I'm not sure about this: if we're pretty sure that a sub-task needs to be run to get the correct output (as we have demonstrated here), do we want it to be disable-able?

Reply:
It would be in keeping with having the option to skip merging positive and negative footprints, for example. I can't think of a situation outside of debugging where we would want it, though. Feel free to leave it out.